net/port.ts

```diff
@@ -0,0 +1,29 @@
+/** @returns an open port */
+export function getAvailable(): number {
+  const tmp = Deno.listen({ port: 0 })
+  const { port } = tmp.addr as Deno.NetAddr
+  tmp.close()
+  return port
+}
+
+/**
+ * Get a promise that resolves upon successful connection to the specified port.
+ * 
+ * @param port the port at which to check readiness
+ * @param ms the duration of the intervals between re-checking readiness
+ * @throws if the inner `Deno.connect` call throws a non `Deno.errors.ConnectionRefused` error.
+ */
+export async function ready(port: number, ms = 500): Promise<void> {
```

Comment:
Let's make the second argument option object. We probably would like to add more options in the future (e.g. `signal: AbortSignal`, `timeout: number`. etc)

ref: https://docs.deno.com/runtime/manual/references/contributing/style_guide#exported-functions-max-2-args-put-the-rest-into-an-options-object

Reply:
I got rid of this function altogether. I think [iuioiua's comment about using `retry`](https://github.com/denoland/deno_std/issues/3279#issuecomment-1836932882) is correct (even moreso in light of mention of abort behavior / ie. would the function throw?).